src/auth.json

```diff
@@ -0,0 +1,3 @@
+{

```

Comment:
we need to make sure this file doesn't go online, because it contains the bot API token (supposed to remain secret)
this file is usually in the gitignore file and therefore is not commited, maybe the spaces at the end of line messed it up.
anyway, the API token was changed.